cfgov/jinja2/v1/youth_employment_success/side-nav.html

```diff
@@ -1,38 +1,36 @@
-                  o-secondary-navigation_list
-                  o-secondary-navigation_list__parents">
-          <li class="m-list_item" data-nav-is-active="False">                  
-            <a class="m-nav-link
-                      m-nav-link__parent" href="/consumer-tools/educator-tools/resources-youth-employment-programs/expand-access-financial-products/">
-                Help Youth Access Financial Products
-            </a>
-          </li>
-          <li class="m-list_item" data-nav-is-active="False">                  
-            <a class="m-nav-link
-                      m-nav-link__parent" href="/consumer-tools/educator-tools/resources-youth-employment-programs/financial-education-resources/">
-                Explore Financial Education Resources
-            </a>
-          </li>
-          <li class="m-list_item" data-nav-is-active="True">                  
-            <a class="m-nav-link
-                      m-nav-link__parent
-                      m-nav-link__current" href="/consumer-tools/educator-tools/resources-youth-employment-programs/transportation-tool/">
-                Transportation Tool
-            </a>
-          </li>
-          <li class="m-list_item" data-nav-is-active="False">
-            <a class="m-nav-link
-                      m-nav-link__parent" href="/consumer-tools/educator-tools/resources-youth-employment-programs/buying-a-car/">
-                Buying a car
-            </a>
-          </li>
-        </ul>    
-      </div>
+      <ul class="m-list
+                m-list__unstyled
+                o-secondary-navigation_list
+                o-secondary-navigation_list__parents">
+        <li class="m-list_item" data-nav-is-active="False">
+          <a class="m-nav-link
+                    m-nav-link__parent" href="/consumer-tools/educator-tools/resources-youth-employment-programs/expand-access-financial-products/">
+              Help Youth Access Financial Products
+          </a>
+        </li>
+        <li class="m-list_item" data-nav-is-active="False">
+          <a class="m-nav-link
+                    m-nav-link__parent" href="/consumer-tools/educator-tools/resources-youth-employment-programs/financial-education-resources/">
+              Explore Financial Education Resources
+          </a>
+        </li>
+        <li class="m-list_item" data-nav-is-active="True">
+          <a class="m-nav-link
+                    m-nav-link__parent
+                    m-nav-link__current" href="/consumer-tools/educator-tools/resources-youth-employment-programs/transportation-tool/">
+              Transportation Tool
+          </a>
+        </li>
+        <li class="m-list_item" data-nav-is-active="False">
+          <a class="m-nav-link
+                    m-nav-link__parent" href="/consumer-tools/educator-tools/resources-youth-employment-programs/buying-a-car/">
+              Buying a car
+          </a>
+        </li>
+      </ul>
     </div>
-  </nav>
-</aside>
+  </div>
+</nav>
```

Comment:
Trivial: Missing EOF newline.

Reply:
Hm pulling up the file, the EOF newline is there.

Reply:
It was added in your latest commit. Did you commit that via the GitHub suggestion mechanism, to fix the class alignment? Maybe it did it automatically when you did.

Reply:
Hm not sure what happened, but on second check I saw and fixed it.